webapp-django/crashstats/settings/base.py

```diff
@@ -659,4 +659,13 @@ def filter(self, record):
 # This is the number of versions to display if a particular product
 # has no 'featured versions'. Then we use the active versions, but capped
 # up to this number.
-NUMBER_OF_FEATURED_VERSIONS = config("NUMBER_OF_FEATURED_VERSIONS", 4)
+NUMBER_OF_FEATURED_VERSIONS = config("NUMBER_OF_FEATURED_VERSIONS", 4, cast=int)
```

Comment:
Given that this wasn't cast and thus was probably wrong, it's probably not used anywhere anymore. We might be able to remove it. But I think I'm going to think about that another day.